src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/register_app_interface_request.h

```diff
@@ -215,11 +224,76 @@ class RegisterAppInterfaceRequest
    */
   bool IsApplicationSwitched();
 
- private:
+  /**
+   * @brief TransformPolicyAppIdToUpper make policy app_id  in msg_params
+   * uppercase.
+   * SDL compare policy appid in not case sensitive mode.
+   */
+  void TransformPolicyAppIdToUpper();
```

Comment:
@LuxoftAKutsan as I could see from implementation you are transforming app id to lower. This function should be renamed